tests/functional/smoke/full-compaction.sh

```diff
@@ -2,9 +2,9 @@
 
 # SPDX-License-Identifier: Apache-2.0
 #
-# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
+# Copyright (C) 2021-2023 Micron Technology, Inc. All rights reserved.
```

Comment:
This file is new with this PR correct?

Reply:
What I am getting at here is this should just be 2023.

Reply:
Ah, cut-n-paste fraud..